app/app.js

```diff
@@ -0,0 +1,61 @@
+import $ from "jquery";
+import * as tournament from "./tournament/tournament";
+import * as player from "./tournament/player";
+
+var mTournament = new tournament.Tournament();
+
+$(document).ready(function() {
+    $("#buttonAdd").click(addPlayerIfValid);
+    $("#buttonCreate").click(generateTournament);
+});
+
+function addPlayerIfValid() {
+    var info = getPlayerInfo();
+
+    try {
+        var newPlayer = new player.Player(info.name, info.rank);
+        mTournament.addPlayer(newPlayer);
+        updateTable(newPlayer);
+
+    } catch (e) {
+        window.alert("Not a valid player");
+    }
+}
+
+function getPlayerInfo() {
+    return {
+        name: $("#fieldName")[0].value,
+        rank: $("#fieldRank")[0].value
```

Comment:
Typically in jQuery you don´t need to access [0]. By using .val() you can get or set the value of the elements, which will work for 1 or more elements.